crp_APL/plan_behavior_planning/src/behaviorPlanner.cpp

```diff
@@ -24,8 +28,15 @@ void crp::apl::BehaviorPlanner::routeCallback(const autoware_planning_msgs::msg:
 
 void crp::apl::BehaviorPlanner::scenarioCallback(const crp_msgs::msg::Scenario::SharedPtr msg)
 {
-    // TODO
-    return;
+    crp_msgs::msg::TargetSpace target_space_msg;
+
+    if (msg->paths.size() > 0)
+        target_space_msg.path = msg->paths[0];

```

Comment:
The speed inside the path should be constrained by the maximum_speed in scenario.